scripts/MKW_TTK.py

```diff
@@ -20,14 +22,15 @@ def onStateLoad(is_slot, slot):
 @event.on_frameadvance
 def onFrameAdvance():
     global playerInputs, ghostInputs
-    frame = core.get_frame_of_input()
-    
+    frame = frame_of_input()
+    in_race = RaceManager.state().value >= RaceState.COUNTDOWN.value
+
     ghostInput = ghostInputs[frame]
-    if (ghostInput and classes.RaceInfo.stage() >= 1):
+    if (ghostInput and in_race):
         TTK_Lib.writeGhostInputs(ghostInput)
     
     playerInput = playerInputs[frame]
-    if (playerInput and classes.RaceInfo.stage() >= 1):
+    if (playerInput and in_race):
```

Comment:
> Applying suggestions on deleted lines is not supported.

Thanks GitHub. Minor nitpick, but we should call this `input_ready` and only allow states 1 and 2.